app/assets/javascripts/all.js

```diff
@@ -5,3 +5,4 @@
 //
 //= require_tree ./
 //= require_tree ./views
+//= require govuk_publishing_components/all_components
```

Comment:
What does this file do compared to `frontend.js`?

Reply:
An excellent question. I have no idea. But I've just checked, and this line doesn't need to be here, but it does need to be in frontend.js.